### PR TITLE
SUIT-10001 Update mocha demo

### PR DESCRIPTION
**Pull request by AntonSBilan**
Update mocha demo

### CODE REVIEW

File: test/dummy.test.js [hunk 1]
```diff
@@ -3,14 +3,8 @@ const {assert} = suitest;
...
-		try {
-			// Start test
-			await suitest.startTest('my-super-cool-test');
-		} catch (e) {
-			console.error(e);
-			// Mocha wouldn't stop if just throw an error here
-			process.exit(1);
-		}
+		// Start test
+		await suitest.startTest('my-super-cool-test');
```
Comment (andriichumak) [inline test/dummy.test.js]: @AntonSBilan , did you tried to make this line fail? Is it really not needed anymore?

Comment (AntonSBilan) [reply test/dummy.test.js]: @andriichumak I tried throw error inside before block, and in this case mocha exits

Comment (andriichumak) [reply test/dummy.test.js]: OK, please try as well with unhandled Promise rejection. Maybe they fixed it in new Mocha version, but when this demo was created there was a problem with it.

Comment (AntonSBilan) [reply test/dummy.test.js]: Promise.reject(''); causes process exit as well

Comment (givehug) [reply test/dummy.test.js]: I tried to throw error in startTest chain explicitly and mocha keeps running.

Comment (givehug) [reply test/dummy.test.js]: ok, its works, probably tested with old version

**Pull request merged**